index.bs

```diff
@@ -699,7 +701,9 @@ When requested, the {{XRSession}} |session| MUST <dfn>apply the pending render s
   1. Let |activeState| be |session|'s [=active render state=].
   1. Let |newState| be |session|'s [=pending render state=].
   1. Set |session|'s [=pending render state=] to <code>null</code>.
+  1. Let |oldLayer| be |activeState|'s {{XRRenderState/baseLayer}}.
   1. Set |activeState| to |newState|.
+  1. If |oldLayer| is not equal to |activeState|'s {{XRRenderState/baseLayer}}, [=update the viewports=] for |session|.
```

Comment:
This is good, but it doesn't account for the possibility that the existing `baseLayer`'s dimensions change. That's an edge case, admittedly, and I don't think you need to block this PR because of it, just something to consider.

Reply:
Fixed